node/defaults.go

```diff
@@ -81,6 +83,40 @@ var DefaultConfig = Config{
 	LogConfig: &LogConfig{
 		TimeFormat: &DefaultTimeFormat,
 	},
+	Storage: &rawdb.StorageConfig{
+		ChainDB: &rawdb.ChainDBConfig{
+			KVDBConfig: rawdb.KVDBConfig{
+				DBType:     "pebble",
+				CacheRatio: 7, // 7% of the total db cache, total cache is 100%
+			},
+		},
+		IndexDB: &rawdb.KVDBConfig{
+			DBType:     "pebble",
+			CacheRatio: 19,
+		},
+		TrieDB: &rawdb.TrieDBConfig{
+			Config: shardingdb.Config{
+				DBType:         "pebble",
+				CacheRatio:     50,
+				Namespace:      "trie",
+				EnableSharding: true,
+				ShardNum:       8,
+				Shards: []shardingdb.ShardConfig{
+					{Indexes: "0-7"},
+				},
+			},
+		},
+		SnapDB: &shardingdb.Config{
+			DBType:         "pebble",
+			CacheRatio:     24,
```

Comment:
do not move // SnapDbResourcePercentage is estimated from on-disk size proportions of snapshot data.
	SnapDbResourcePercentage = 24,  set it as default value here , use const value . the same applies to the other databases as well